src/Chartjs/DependencyInjection/ChartjsExtension.php

```diff
@@ -38,9 +40,10 @@ public function load(array $configs, ContainerBuilder $container)
             ->setPublic(false)
         ;
 
-        if (class_exists(Environment::class)) {
+        if (class_exists(Environment::class) && class_exists(StimulusTwigExtension::class)) {
```

Comment:
Using `willBeAvailable()` would be nice, but it's not available until 5.3 https://github.com/symfony/symfony/blob/e9a702635a89e84d2c63f25d066e63d729017741/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L1460